cap/modules/auth/views.py

```diff
@@ -76,8 +76,14 @@ def inner(*args, **kwargs):
 @login_required
 @auth_error_handler
 def connect(name):
-    next_url = request.args.get('next')
-    ui_flag = request.args.get('ui')
+    try:
+        next_url = request.args.get('next')
+        ui_flag = request.args.get('ui')
+    except Exception:
```

Comment:
should have explicit exception, we should avoid general general `Exception`

Reply:
okay, We are doing `.get` so there will be no error here(tested again), while testing I found that `name` will be some oauth service and on invalid service name in `connect/<name>`, it throws 500. I added a check next to it to check the services coming from [config](https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/master/cap/modules/auth/config.py#L49) else it will raise `RESTException`